src/varity/chain.clj

```diff
@@ -61,41 +61,62 @@
   "Loads f (e.g. hg19ToHg38.over.chain(.gz)), returning the all contents as a
   sequence."
   [f]
-  (with-open [^java.io.Reader rdr (try (-> (CompressorStreamFactory.)
-                                           (.createCompressorInputStream (io/input-stream f))
-                                           io/reader)
-                                       (catch CompressorException _ (io/reader f)))]
+  (with-open [rdr (io/reader (cljam.util/compressor-input-stream f))]
```

Comment:
`cljam.util/compressor-input-stream` is a general utility function and is not peculiar to cljam, so using it from other project through cljam is somewhat strange. If in Java, I made it package private. I think it should be reimplemented as `varity.util/compressor-input-stream`.